ceph-releases/ALL/ubi8/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -1,2 +1,14 @@
 microdnf update -y --setopt=install_weak_deps=0 --nodocs && \
-microdnf install -y --setopt=install_weak_deps=0 --nodocs wget unzip util-linux python3-saml python3-setuptools udev device-mapper __CEPH_BASE_PACKAGES__
+microdnf install -y --setopt=install_weak_deps=0 --nodocs wget unzip util-linux python3-saml python3-setuptools udev device-mapper __CEPH_BASE_PACKAGES__ && \
+ln -s /usr/share/ceph/mgr/dashboard/frontend/dist-ibm /usr/share/ceph/mgr/dashboard/frontend/dist && \
+sed -i \
+  -e "s|registry.redhat.io/rhceph/rhceph-5-rhel8:latest|icr.io/ibm-ceph/ceph-5-rhel8|" \
+  -e "s|registry.redhat.io/openshift4/ose-prometheus:v4.10|icr.io/ibm-ceph/prometheus:v4.10|" \
+  -e "s|registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.10|icr.io/ibm-ceph/prometheus-node-exporter:v4.10|" \
+  -e "s|registry.redhat.io/rhceph/rhceph-5-dashboard-rhel8:latest|icr.io/ibm-ceph/ceph-5-dashboard-rhel8:latest|" \
+  -e "s|registry.redhat.io/openshift4/ose-prometheus-alertmanager:v4.10|icr.io/ibm-ceph/prometheus-alertmanager:v4.10|" \
+  -e "s|registry.redhat.io/rhceph/rhceph-haproxy-rhel8:latest|icr.io/ibm-ceph/haproxy-rhel8:latest|" \
+  -e "s|registry.redhat.io/rhceph/keepalived-rhel8:latest|icr.io/ibm-ceph/keepalived-rhel8:latest|" \
+  -e "s|registry.redhat.io/rhceph/snmp-notifier-rhel8:latest|icr.io/ibm-ceph/snmp-notifier-rhel8:latest|" \
+  -e "s|default='registry.redhat.io'|default='icr.io'|" \
+  /usr/share/ceph/mgr/cephadm/module.py
```

Comment:
are we going to do this for the cephadm binary as well (`/usr/sbin/cephadm`)? It's probably not strictly necessary for things to work, but the version in the container will be getting deployed on the host by the mgr module.

Reply:
Yes, we edit the `/usr/sbin/cephadm` binary separately (during the `cephadm` RPM build). The reason we do that before we install the `cephadm` RPM into the container is that since we must distribute that `cephadm` RPM separately for users to bootstrap a cluster.

(Incidentally this last `-e "s|default='registry.redhat.io'|default='icr.io'|"` substitution is slightly different for `module.py` compared to `cephadm`, where we have to do `-e "s|DEFAULT_REGISTRY = 'registry.redhat.io'|DEFAULT_REGISTRY = 'icr.io'|"`)